home-page/style.css

```
@@ -0,0 +1,201 @@
+/* Gernral Properties */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body {
+    font-family: "Roboto", sans-serif;
+}
+.container {
+    width: 80%;
+    margin: auto;
+}
+a {
+    color: #000;
+    text-decoration: none;
+}
+/* start section showcase */
+.showcase-fluid {
+    background-image: url('../images/home/background-showcase.jpg');
+    background-size: cover;
+    background-position:center;
+    background-repeat: no-repeat;
```

You can try something like this and you can even increase the black transparent background
```suggestion
    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
    url("../images/home/background-showcase.jpg") no-repeat center center;
    background-size: cover;
```

---

I did this now